packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.tsx

```diff
@@ -13,13 +13,40 @@ export class SwirlTableRowGroup {
 
   @Prop() label!: string;
   @Prop() tooltip?: string;
+  @Prop() collapsable?: boolean = false;
+
+  @State() isVisible: boolean = true;
+
+  private toggleShowItems = () => {
+    const rowsContainer = this.el.shadowRoot.querySelector(
```

Comment:
Suggestion: Use the JSX ref attribute to get the element reference. https://stenciljs.com/docs/templating-jsx#getting-a-reference-to-a-dom-element

Reply:
Yes, that works. Cool, thanks!